helm/ph-ee-engine/values.yaml

```diff
@@ -85,7 +85,7 @@ camunda-platform:
       - name: init-ph-ee-kafka-exporter
         image: busybox:1.28
         command: ['/bin/sh', '-c']
-        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://fynarfin.io/images/exporter-1.1.1-SNAPSHOT.jar"; ls -al /exporters/']
+        args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://fynarfin.io/images/exporter-3.0.0-SNAPSHOT.jar"; ls -al /exporters/']
```

Comment:
if the backward compatibility is not broken, increase the minor version

Reply:
Updated the version.